version.go

```diff
@@ -11,10 +14,18 @@ const VersionMinor = 0
 // VersionPatch holds the release patch number
 const VersionPatch = 0
 
-// Version holds the combination of major minor and patch as a string
-// of format Major.Minor.Patch
-var Version = fmt.Sprintf("%d.%d.%d", VersionMajor, VersionMinor, VersionPatch)
```

Comment:
if we only delete `fmt.Sprintf(xxx)`, we can not show the Major minor version of containerd commands, only git commit.
so I suppose if  the version string like this: ( just like runC)
`
VERSION:
   1.0.0
commit: b160a7b.m

`